app.js

```diff
@@ -0,0 +1,46 @@
+var express = require('express');
```

Comment:
Change these `var`s to `const`.

Use `const` instead of `var` if you don't plan on reinitializing the variable you're declaring. Use `let` if you do.